index.js

```diff
@@ -418,6 +418,12 @@ class HyperBee {
     return iteratorToStream(new DiffIterator(new Batch(this, null, false, opts), new Batch(right, null, false, opts), opts), active)
   }
 
+  async hasPrefix (prefix) {
+    const ite = this.createRangeIterator({ gte: prefix })
+    await ite.open()
+    return ite.next()
```

Comment:
capture the node here and check that the key actually starts with the prefix

Reply:
ie, 'z' is gte 'a' but 'z' doesn't have 'a' as a prefix

Reply:
corrected